requirements.txt

```diff
@@ -52,7 +52,7 @@ ndg-httpsclient==0.3.2
 networkx==1.7
 pathtools==0.1.2
 poster==0.8.1
-psycopg2==2.4.5
+psycopg2==2.7.4
```

Comment:
I'd change that to `psycopg2-binary==2.7.4`
http://initd.org/psycopg/articles/2018/02/08/psycopg-274-released/